packages/components-js/tests/e2e/puppeteer/helpers/events.ts

```diff
@@ -26,7 +26,7 @@ export const addEventListener = (handle: ElementHandle, eventName: string): Prom
         id: target.id,
       };
       el[counterKey] = (el[counterKey] || 0) + 1;
-      el[detailsKey] = [...el[detailsKey], detail];
+      el[detailsKey] = [...(el[detailsKey] || []), detail];
```

Comment:
Not sure if this is still needed but shouldn't it be needed on the following line, too?